.github/workflows/release_deployments.yml

```diff
@@ -1,27 +1,50 @@
 name: Deploy on Release
```

Comment:
It seems that almost all GH actions and workflows use a naming convention, with all lowercase and where hyphens are used instead of underscores. 

If we adhere to that you should rename `release_deployment.yml` into `release-deployment.yml`.